src/api.ts

```diff
@@ -272,7 +272,10 @@ export function fernsRouter<T>(
       // TODO we can make this much more complicated with ands and ors, but for now, simple queries
       // will do.
       for (const queryParam of Object.keys(req.query)) {
-        if ((options.queryFields ?? []).concat(SPECIAL_QUERY_PARAMS).includes(queryParam)) {
+        if (SPECIAL_QUERY_PARAMS.includes(queryParam)) {
+          continue;
```

Comment:
Since the rest of this for loop is not wrapped in an else, what is the point of having this `if` with `continue`? Won't it continue if `true` or `false`?

Reply:
I think I'm confused. If the queryParam is in the special query params list, it hits `continue` and skips the rest of the execution, moving to the next queryParam in the for loop. Otherwise it decides whether to add the the param to the actual query or throw an error if it's not an allowed param. At least, that's what it's supposed to do!

Reply:
It's not you that's confused, it's me 😆 . This looks good and makes sense!

Reply:
All good! Monday after a long weekend!